Assets/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "com.amplitude.unityplugin",
```

Comment:
We have constant defined in our c# code. Is there a way to fetch the version number from that place?

Reply:
I will think of a way to automate this in the future. This is a .json file so some sort of script?